tests/common/gu_utils.py

```diff
@@ -36,8 +36,17 @@ def delete_tmpfile(duthost, tmpfile):
     duthost.file(path=tmpfile, state='absent')
 
 
-def format_json_patch_for_multiasic(duthost, json_data, is_asic_specific=False):
-    if is_asic_specific:
+def format_json_patch_for_multiasic(duthost, json_data, is_asic_specific=False, is_host_specific=False):
+    """
+    Formats a JSON patch for multi-ASIC platforms by adding a path change to be applied
+    simultaneously in all DUT host namespaces (localhost + ASIC namespaces).
+
+    - For changes that are ASIC-specific (applied only in one ASIC namespace), the function
+    returns without any formatting.
+    - For changes that are host-specific (applied only to the localhost namespace and not
+    to ASIC namespaces), the function returns without any formatting.
+    """
+    if is_asic_specific or is_host_specific:
```

Comment:
>if is_asic_specific or is_host_specific: [](http://example.com/codeflow?start=4&length=40)

From code, these two variables should be no difference between asic and host, per discussed offline, we could add an variable that indicates the patch contains namespace already.

Reply:
Based on similar logic as the flag `is_host_specific`, which makes zero changes to the patch, I have added a new flag `is_asic_specific` to again ensure zero changes to the patch.

In such cases, there is an assumption that the function `format_json_patch_for_multiasic` has already been called with the namespace included in the patch path. This must be ensured by each test suite file that invokes this function.